_extras/customization.md

```diff
@@ -91,63 +99,50 @@ which must define the following values in its header:
 
 The header may optionally define the following:
 
-*   `etherpad` is the URL for the Etherpad for your workshop.  If you are
+*   `collaborative_notes` is the URL for the Etherpad for your workshop.  If you are
     not using an Etherpad, you can delete this line. You can create a carpentries etherpad [here](https://pad.carpentries.org/).
 
 *   `eventbrite` is the multi-digit Eventbrite registration key.  If you
-    are using Eventbrite, the Software Carpentry administrators will
+    are using Eventbrite, the Carpentries Regional Coordinators will
     give this to you.  If you are using something else, you may delete
     this line.  Note: this value must be given as a string in double
     quotes, rather than as a number.
 
+### For online workshops
```

Comment:
@maneesha, @zkamvar, @ErinBecker could you please review the paragraph below.

Reply:
This makes sense and looks good to me